ee/ucp/admin/configure/join-nodes/use-a-load-balancer.md

```diff
@@ -34,6 +34,8 @@ or port number.
     * Use the same load balancer with multiple virtual IPs.
 * Configure your load balancer to expose UCP or DTR on a port other than 443.
 
+> **Note:** In addition to configuring your load balancer to distinguish between UCP and DTR, configuring a load balancer for DTR has [additional requirements](https://docs.docker.com/ee/dtr/admin/configure/use-a-load-balancer/#load-balance-dtr).
+
```

Comment:
1. Perhaps this should be an important admonition since it can cause it not to work: ee/ucp/admin/configure/join-nodes/use-a-load-balancer.md

2. This links should be relative since it is on the same site.

Reply:
@ahh-docker will reformat.